package.json

```diff
@@ -66,6 +66,12 @@
       "typescript"
     ],
     "jest": {
+      "globals": {
+        "caches": true
+      },
+      "setupFiles": [
+        "<rootDir>/test/setupTests.ts"
```

Comment:
I think ideally we'd fix [`build-config` to resolve `.ts` setup files](https://github.com/apache-superset/build-config/blob/master/configs/jest.js#L12) but this is okay for now.